src/sage/tests/books/computational-mathematics-with-sagemath/integration_doctest.py

```diff
@@ -253,7 +253,7 @@
 Sage example in ./integration.tex, line 1363::
 
   sage: t, y = var('t, y')
-  sage: desolve_rk4(t*y*(2-y), y, ics=[0,1], end_points=[0, 1], step=0.5)
+  sage: desolve_rk4(t*y*(2-y), y, ics=[0,1], end_points=[0, 1], step=0.5) # abs tol 1e-12
   [[0, 1], [0.5, 1.12419127424558], [1.0, 1.461590162288825]]
```

Comment:
Please two spaces `...)  # abs ...` instead of one `...) # abs ...`

Reply:
Sure, done.